api/v1alpha1/imagepolicy_types.go

```diff
@@ -53,6 +56,18 @@ type SemVerPolicy struct {
 	Range string `json:"range"`
 }
 
+// AlphabeticalPolicy specifices a alphabetical ordering policy.
+type AlphabeticalPolicy struct {
+	// Prefix enables filtering the list of tags by given prefix.
+	// +optional
+	Prefix string `json:"prefix,omitempty"`
```

Comment:
Is this a pattern or a literal? Does the matching prefix get removed before ordering? (Worth putting these considerations in the docs!)

Reply:
Right now it's just a literal, using `strings.HasPrefix`. We can discuss the possibility to use a pattern, though I wasn't able to think of any use case for that, however, some people might find support for that useful. What do you think?

As a side note, zooming out a bit, I think it might make sense to provide the ability to filter out tags at the level of the imagerepository resource as well, in order to prevent polluting the tags database with any unused entries.

Reply:
I think you need to clarify the intent of the prefix, i.e. is it matching, or excluding.

Reply:
I would think users might want to limit the tags the policy would apply to only to a subset of tags e.g `rel-*` or `nightly-*`. The intention was to use `Prefix` for matching this subset, excluding other irrelevant tags.

Would renaming this attribute to `MatchTagPrefix` make more sense?
What about supporting multiple values?
Or maybe adding support for regular expressions?

Reply:
Made some changes, however, some of the questions I mentioned above are still valid.